kitsune/questions/config.py

```diff
@@ -34,37 +34,37 @@
                         # TODO: Just use the IA topics for this.
                         # See bug 979397
                         (
-                            "download-and-install",
+                            "Install and Update",
                             {
-                                "name": _lazy("Download, install and migration"),
-                                "topic": "download-and-install",
-                                "tags": ["download-and-install"],
+                                "name": _lazy("Install and Update"),
```

Comment:
I think the `Update` should be `update`? So:
```suggestion
                                "name": _lazy("Install and update"),
```